portfolio/src/main/webapp/script.js

```diff
@@ -12,31 +12,32 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+  const greetings = [
+    'Hello world!', 
+    '¡Hola Mundo!', 
+    '你好，世界！', 
+    'Bonjour le monde!'
+  ];
 
-//   Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
 
-//   Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
 
 function addRandomFact() {
-    const facts = 
-    ['I have 2 dogs, Olive and Bella', 'I am turning 20 on June 30th', 'I live on a farm',
+  const facts = [
+    'I have 2 dogs, Olive and Bella', 
+    'I am turning 20 on June 30th', 
+    'I live on a farm',
     'I switched my major from Biomedical Engineering to CS at the end of my first year',
-    'I have never been to the West Coast', 'My dogs have an instagram: @olive_and_bella'];
+    'I have never been to the West Coast', 
+    'My dogs have an instagram: @olive_and_bella'
```

Comment:
nit: I think this needs a comma at the end (same for the greetings above)

https://google.github.io/styleguide/jsguide.html#features-arrays-trailing-comma